pkg/baremetal/utils/raid/megactl/megactl.go

```diff
@@ -118,7 +118,11 @@ func (dev *MegaRaidPhyDev) parseLine(line string) bool {
 		} else if val == "Rebuild" {
 			dev.Status = "rebuild"
 		} else {
-			dev.Status = "offline"
```

Comment:
Firmware states are either "Failed", "Online, Spun Up", "Online, Spun Down", "Unconfigured(bad)", "Unconfigured(good), Spun down", "Hotspare, Spun down", "Hotspare, Spun up" or "not Online".

Reference: https://cs.uwaterloo.ca/twiki/view/CF/MegaRaid

Reply:
1.If a "Hot Spare" was present the hot spare will now copy its data to the new drive. The alarm may sound, the blue LED should be on, and the red LED flashing indicating a "**copy back**". As well you should notice all other drives flashing.
2.The JBOD enclosure may not have all drives assigned to the array. An unused drive will show as "**Unconfigured(good), Spun down**".
It may require several minutes (maybe half an hour) for the array controller to recognize a new drive install.
3. **Online, Spun down**:A faulty drive may not go into the off-line mode. Check to see if the array is in the "spun down" state. If it is spun down then access or edit a file in the array. That will then spin up the drives. Then try to "off-line" the faulty drive.  If the drive won't go off-line then just go ahead with the drive replacement. However, note that the new drive may not be immediately recognized by the adapter. It may require up to half an hour for the controller to acknowledge the new drive.
Check the drive status again. It may show that the drive no longer exists in the array or that it is still faulty. After waiting half an hour you may want to re-seat the drive

Reply:
4.**Unconfigured(bad)**:Take the faulty drive offline. It may start the alarm once offline, so stop the alarm.